tools/xmlGenerator/domainGenerator.py

```diff
@@ -209,7 +209,9 @@ def main():
                             args.schemas_dir],
                            stdout=sys.stdout, stdin=subprocess.PIPE, stderr=sys.stderr)
 
-    initial_settings = os.path.realpath(args.initial_settings)
+    initial_settings = ""
```

Comment:
"keep" implies that it has another value beforehand, which it doesn't.

The point of this change is that of args.initial_settings isn't set, it musn't be passed to `os.path.realpath()`. But we need `initial_settings` to be exist regardless since we are passing to as argument to `generateDomainCommands()` below. `""` evaluates to `False`, so it seems sensible to indicate an absence of value; but I understand your point.